docs/_build/html/_sources/tutorial.rst.txt

```diff
@@ -0,0 +1,198 @@
+####################
+ CytoSnake Tutorial
+####################
+
+*******
+ About
+*******
+
+This tutorial assumes that you have follows the installation steps and
+you are ready to start taking off with CytoSnake!
+
+Cytosnake is a command line interface (CLI) tool that contains a
+multitude of workflows for analyzing morphological features obtained
+from microscopy images of cells.
+
+**********
+ Concepts
+**********
+
+.. image:: images/Cytopipe-cli_0.drawio.png
+
+Modes
+=====
+
+Modes provide options on how the user can change the functionality of
+CytoSnake. For example, if you would like to initialize your files for a
+specific workflow, you can simply type:
+
+using the `init` mode
+
+.. code::
+
+   cytosnake init <DATAFILES> <METADATA> <DATATYPE>
+
+-  `DATAFILE` will refer to the raw data that you are going to analyze,
+-  `METADATA` refers to the associated metadata data directory that was
+   generated along with the dataset
+-  `DATATYPE` flag tells cytosnake weather these morphology feature
+   datasets were obtained from CellProfiler or DeepProfiler
+
+The init mode setups the provided input files into its appropriate file
+structure that accommodates all the workflows available in Cat Snake
+
+Currently, cytosnake supports three different types of modes, which are:
+
+   #. inti: setups up input files for workflows
+   #. run: execute a specific
+   #. help: executes CytoSnake's CLI help documentation.
+
+Configurations
+==============
+
+CytoSnake has configurational directory that allows users to change the
+configurations for their specified workflows.
+
+The configuration files are written in `.yaml` files, which contains all
+the functions and its parameters used within the workflow. The
+workflow's documentation provides information about the configuration
+files involved within the workflow.
+
+Users can easily find and change parameter values by accessing those
+configurational files.
```

Comment:
```suggestion
CytoSnake has a config directory which stores config files.
Users can modify these configs for their custom workflows.

The config files are written in `.yaml` , and they contain all functions and  parameters used in data processing workflows.
 Each file provides documentation about the configuration
files involved within the workflow.

Users can easily find and change parameter values by modifying config files.
```

I made a couple edits for clarity. Please confirm their accuracy!ss

Reply:
Yes this is accurate, thank you.